storm-server/src/main/java/org/apache/storm/daemon/supervisor/Slot.java

```diff
@@ -1147,8 +1162,19 @@ public DynamicState withPendingLocalization(Future<Void> pendingDownload) {
          */
         public DynamicState withState(final MachineState state) {
             long newStartTime = Time.currentTimeMillis();
+            //TODO: potential lost metrics due to timing accuracy (Timer only tracks one call per millisecond)
```

Comment:
Please don't leave TODOs int he code.  Either fix it, file a follow on JIRA to fix it, or accept it and just have it be a comment and not a TODO.